docs/book/setup/installing-sylius-plus.rst

```diff
@@ -1,8 +1,7 @@
-
 .. rst-class:: plus-doc
 
-Sylius Plus Installation
-========================
+How to Install Sylius Plus?
```

Comment:
why are you changing this?
We are in the book, we should not use the "How to" format, this is for cookbooks :D

And if you have to then at least:
```suggestion
How to install Sylius Plus?
```

Reply:
Mostly to make it look more like a cookbook - it's a how-to article, step-by-step guide for Sylius Plus installation, without explaining the details of the process. I'm OK with reverting it and I'll do that (so that we can tackle it independently and once for all in the future).

As for capitalising, I follow title capitalisation rules (https://capitalizemytitle.com/).

Reply:
Anyway, my end goal is to get rid of cookbooks and embed them in topic-related book articles. Eg. looking at "Resource models" article and having "How to customise a model?", "How to make model translatable?" articles linked at the bottom.

Reply:
But these are sentences not "titles" like for blog posts. And sentences are written like sentences, it's a common practice. We have never used title-case ever before and I would not introduce it just because we found out it exists :).

Reply:
That's why I reverted this change - not to add irrelevant changes to the docs unification PR.

I could argue whether it is a title or not (well, it's an article after all) and whether never using title case is a good enough argument to refrain from using them. 

Our docs contain a recommendation for section names, https://docs.sylius.com/en/latest/book/contributing/documentation/standards.html#language-standards, so we should define it there.